aws-bundle/NOTICE

```diff
@@ -9,57 +9,81 @@ The Apache Software Foundation (http://www.apache.org/).
 
 NOTICE for Group: commons-codec  Name: commons-codec  Version: 1.17.1
 
-src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
-contains test data from http://aspell.net/test/orig/batch0.tab.
-Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)
+| Apache Commons Codec
+| Copyright 2002-2024 The Apache Software Foundation
+| 
+| This product includes software developed at
+| The Apache Software Foundation (https://www.apache.org/).
```

Comment:
Now that this is a boilerplate NOTICE, I think we may want to remove it per the [recommendation from Roy](https://github.com/apache/iceberg-python/commit/fbdf04b0f4d83719b26facbac25b4695ff8f0629#commitcomment-140697012).

Reply:
Yes, agree. I did that for "consistency". I propose to done a new PR after the releases but this change.

Reply:
Done !